nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-controller-service/pom.xml

```diff
@@ -91,4 +107,18 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <id>kudu-windows</id>
```

Comment:
You might need to move the `kudu-binary` dependency into a profile for OSX and Linux so that Maven doesn't try to resolve it on Windows, which will result in a missing dependency.  ex:
```
<profile>
      <id>kudu-linux</id>
      <activation>
        <os>
          <family>Unix</family>
        </os>
      </activation>
      <dependencies>
        <dependency>
          <groupId>org.apache.kudu</groupId>
          <artifactId>kudu-binary</artifactId>
          <version>${kudu.version}</version>
          <classifier>${os.detected.classifier}</classifier>
          <scope>test</scope>
        </dependency>
      </dependencies>
    </profile>
    <profile>
      <id>kudu-mac</id>
      <activation>
        <os>
          <family>mac</family>
        </os>
      </activation>
      <dependencies>
        <dependency>
          <groupId>org.apache.kudu</groupId>
          <artifactId>kudu-binary</artifactId>
          <version>${kudu.version}</version>
          <classifier>${os.detected.classifier}</classifier>
          <scope>test</scope>
        </dependency>
      </dependencies>
    </profile>
```